distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Widget_Helpers.enso

```diff
@@ -156,18 +162,23 @@ make_join_kind_selector display=Display.Always =
 
 ## PRIVATE
    Make a join condition selector.
-make_join_condition_selector : Table -> Display -> Widget
-make_join_condition_selector table display=Display.Always =
-    col_names_selector = make_column_name_selector table display=Display.Always
+make_join_condition_selector : Table -> Display -> Function -> Widget
+make_join_condition_selector table display=Display.Always cache=Nothing =
+    right = if cache.is_nothing.not then cache "right"
```

Comment:
This is OK, but I recommend using an 'idiom' we have for computing something if a value is not nothing:

```suggestion
    right = cache.if_not_nothing <| cache "right"
```